app/models/miq_server.rb

```diff
@@ -142,9 +107,12 @@ def start
     clear_miq_queue_for_this_server
 
     #############################################################
-    # Call all the startup methods only NOW, since some check roles
+    # Other startup actions
```

Comment:
Turns out that *none* of the atStartup methods check roles, so this message is a) basmati rice and b) invalid in that the startup methods can be run whenever.